sample-docs/kitchen-sink/api.rst

```diff
@@ -25,5 +25,5 @@ Using Sphinx's :any:`sphinx.ext.autodoc` plugin, it is possible to auto-generate
         # Don't show class signature with the class' name.
         autodoc_class_signature = "separated"
 
-.. automodule:: urllib.parse
+.. automodule:: code
```

Comment:
Actually, this is no better than urllib.

```console
lib/python3.12/code.py:docstring of codeop.compile_command:7: ERROR: Unexpected indentation. [docutils]
lib/python3.12/code.py:docstring of codeop.compile_command:8: WARNING: Block quote ends without a blank line; unexpected unindent. [docutils]
```

Switching to `bz2` seems to work OK.